jubilant/_test_helpers.py

```diff
@@ -16,6 +16,16 @@ def temp_model(keep: bool = False, controller: str | None = None) -> Generator[J
 
     Provides a :class:`Juju` instance to operate on.
 
+    If you want to configure ``Juju()`` parameters such as ``wait_timeout``, set the appropriate
```

Comment:
There's a reference to the object Juju right above, so maybe this line can be simplified to:

```suggestion
    If you want to configure its parameters, such as ``wait_timeout``, set the appropriate
```